### ISSUE TITLE
Spotify-tui is not available in Slovenia

### ISSUE THREAD

**Issue by korpseluv** [Rigellute/spotify-tui#1118]
So, I have been enjoying the app but today i was greeted with it not being available in my country or region. Is there a quick fix that I can do or does it require and update to the app?

**Comment by stanekondrej** [comment]
I don't think that is an issue with spotify-tui.. Could you post the error message, or something like a screenshot?

**Comment by korpseluv** [comment]
Sorry for the wait here is the screenshot. Whenever the problem is not there i cant select the output device.
<img width="744" alt="Screenshot 2024-02-13 at 2 14 04 PM" src="https://github.com/Rigellute/spotify-tui/assets/116295241/e110d8a6-d694-4ff5-aefe-57fe9354ff8e">

**Comment by stanekondrej** [comment]
Oh, sorry, judging from the screenshot this actually is a spotify-tui bug. However, it looks like spotify-tui is unmaintained at the moment (last release was 2 years ago). I currently use [spotify-player](https://github.com/aome510/spotify-player), and I recommend you check it out as well, because it has more features, similar keybinds and is maintained. Hope this helps.

**Comment by korpseluv** [comment]
> Oh, sorry, judging from the screenshot this actually is a spotify-tui bug. However, it looks like spotify-tui is unmaintained at the moment (last release was 2 years ago). I currently use [spotify-player](https://github.com/aome510/spotify-player), and I recommend you check it out as well, because it has more features, similar keybinds and is maintained. Hope this helps.

I will definitely check it out, is there any possibility that this particular project will get updated. I honestly like it and I was about to fund it. Not sure but thanks for now!

**Comment by stanekondrej** [comment]
I have no clue, as far as I'm aware the maintainers didn't provide any info on why there are no updates. I'm just a regular user like you.
